person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..500)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
Please use method `of_age?` here as `is_of-age?` is not defined in the file.

Reply:
Thanks @SajjadAhmad14 but `rubocop` keeps flagging it as an offence and said it should be changed to `of_age`

Reply:
Hi @AnselemOdims so you need to use `#of_age?` inside `can_use_services?`.